### ISSUE TITLE
Project / Quality Status?

### ISSUE THREAD

**Issue by prologic** [someone1/zfsbackup-go#3]
Is this project still considered "work in progress"? Who currently uses this to successfully perform remote backups of their ZFS pools in any serious form? Thanks!

**Comment by someone1** [comment]
I've been using it as my primary backup solution for months now - haven't really had time to add new features but hope to soon.

**Comment by prologic** [comment]
Can we keep this open and use this as a way to collect usages and users of
this tool?


James Mills / prologic

E: prologic@shortcircuit.net.au
W: prologic.shortcircuit.net.au

**Comment by someone1** [comment]
Sure, not a problem. I've personally uploaded thousands of snapshots and periodically test my backups to restore to a random point in time and validate the results. I have yet to run into any issues in that regard through the schedule and complexity of my backups.

**Comment by prologic** [comment]
@someone1 Re your previous comment here; would you mind documenting your backup/restore/verification workflow in a Wiki page? That would be super handy to learn from what you've done. (*I don't want to pollute the project with too many open issues!*)

**Comment by someone1** [comment]
Yes I'd be happy to when I find some time. I've been meaning to expand documentation on usage through the Wiki feature anyway, adding use-cases/examples is a great idea.

**Comment by auberginepop** [comment]
Interesting project!
Am I write in saying that with no delete feature the size of your remote storage is always increasing. You don't have the Duplicity features of specifying regular full backups and how many full/incremental backups to keep?

**Comment by jsirianni** [comment]
This issue was opened a while ago, what are the feelings in 2020? I will try and kick the tires on this soon. 

I currently depend on rclone to manage terrabytes of backups to B2, among 8-12 systems.

**Comment by someone1** [comment]
The current software works well for my company's production use so I'm supporting this project through my personal time. I do squash bugs ASAP but my track record for adding new features isn't stellar (apologies)!

I hope to put a new release out soon with a few features that have been more/less ready, add more testing, and close out some of the other open issues.

**Comment by scotte** [comment]
I understand the current project status, and I know I can always build it myself, but a new release with features added since 2020 such as ssh support would be swell. Thanks! :-)
